tap_sftp/client.py

```diff
@@ -51,6 +63,10 @@ def __try_connect(self):
                 self.transport.connect(username= self.username, password = self.password, hostkey = None, pkey = None)
                 self.sftp = paramiko.SFTPClient.from_transport(self.transport)
             self.__active_connection = True
+            # get 'socket'
```

Comment:
Please elaborate more, why do we need to get a socket?

Reply:
To set the timeout we need socket.